test/common.js

```diff
@@ -0,0 +1,147 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import crypto from 'crypto';
+import fs from 'fs';
+import path from 'path';
+import { spawn } from 'child_process';
+import sinon from 'sinon';
+
+const
+	DEFAULT_HASH_ALGORITHM = 'sha256',
+	DEFAULT_HASH_ENCODING = 'hex',
+	TEST_CACHE_PATH = 'test/sandbox/test-cache',
+	TEST_CLIENT_ID = '982f0d7f23a94811849f703ef8150401',
+	TEST_TEMP_PATH = 'test/sandbox/test-temp',
+	TEST_SHARED_SECRET = '68JLP0L6M10/kfvtXNQCrbXe3nz8g1smyCjTblcy4OKbd4cnxf0anuxvLZ0T4hZw8hMi8w8pPvezd6AcBwa8Hw==';
```

Comment:
It goes without saying - remove these credentials and everything else that is PlayNetwork proprietary or that references the `music-nodejs` app (presumably where these tests were copied from) entirely.